CommandHelper/LocalPackages/commands/chars/joker.yml

```diff
@@ -2,10 +2,25 @@ class: joker
+  val: eyJ0aW1lc3RhbXAiOjE1MjQ0MjI4NTUzMDQsInByb2ZpbGVJZCI6ImVmN2JjYzk2M2FjMDRkYzZhYWU3MDE5ZTVjNmRkYTJlIiwicHJvZmlsZU5hbWUiOiJzdXBlcmlvcnhtYW4iLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlL2ExMTdkYWM4YjQ2ZWEyYjhiOTJiYzRlZjZmNmUzZWFkMWI5YWE0MTViYjRiMjI3MTE5NzVmYzI1Yzc0MzM0In19fQ==
+  sig: wxt0ByI0Ja+rWnf0jnaqNwIuStVu6sIuOyro7qSRhKW6FFCoEFyu/0HkvhrnKD+dGD1KQGCxtFTWCONx3u4ZYu78VmX/T1ZaMCdJgy+ksHg8S6KLGEA13ckluCCKs0Em+ZkjXHH4uFCvjvZkwQhkW7ZNIbKQGetpv3sWrhQQzv6CLZAk4zuI69d3GH0JP3mbCWlOxu+9TffExQSIgFa9eJPR+bpnUrewyJcNtANR6rHnWFC/E5mcOW9e1CATUMlkewa1AueL0jcoV3uR2NHAskTsv/z9xMTf2WeJAbi3BMLsYwdKXvY9UZBcsL/yAFyjwCs286pIw0ltrMIKsi0Jn60Y4lOKG4yibnsYsjnLQ0rsT1zwweD9I3O0XC9+d9pBfIM43jUQJ3GqDBypJKjnF7HMPfChnv+WpDXFfQqTfW1ORfR3IYOHsFZ7auPWmpv0TVD+27+buXRPjlIDZf/BdTiKk8KUVznkbnzOIWsFTSI6iaE+df8m4W/8pK0vs3mvD9MtkcRHIYH6GTJI0ml4wNt0TMm07bT8xaexJYQEN25dif3piw9BMt2xloEenr0eB41J00ZNH3QVmmWVYWCkGpQDpKOAb/Mao3lwUNZbnBylZCeg7BDGLdhHHN6UNr7P9Ww0eFh5oJ3E7+Oyw9cx6TBMkeEYdahaBQ9Q+OeOqTU=
+  val: eyJ0aW1lc3RhbXAiOjE1MjQ0Mjc1MDEwMzMsInByb2ZpbGVJZCI6ImVmN2JjYzk2M2FjMDRkYzZhYWU3MDE5ZTVjNmRkYTJlIiwicHJvZmlsZU5hbWUiOiJzdXBlcmlvcnhtYW4iLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzcyOTZlZTk1ODE3ZWYyMzNhM2UwNGY0YjJmNTAxNTJkOGFlMTgyODY4OGY5NDY1ZWUwZGViODM4ODFjNTNiMmEifX19
+  sig: Hmz7Cj0AbWsH5Ow7y/TrpiyBMA4wEF0Ccj3vGBY04SGcTNZTYn103O5++59mWNYHFCJgBtk0/EwGkmcD+Y1oiSTdbjeiFZcqaZQboerVjyFy9kZ2b97M/VNkYTT3YzHXhJuWw2dCUp69QRRBD8Ze6KS82IjShCS4dl4lfFBf/MtmOCJLcoV6hVYCCSwkLB5HL+ar+bko3zB6aWq39ntOLo9sPJqGcP8tCRm48FYdPgnL3kjFOT1mXorloyOhBA1I6WZ7U+DKl0y/JwJLBMzwJzpeo2OVpn3RxPyghQkapFViQmydT0RVYOkYE4/tLkDeRKsMYK0rDgTYkdFoP71YUOaEQeudvz1JEj7f/T6cvKC5nymVaPjnnH35cFKvA+016AZa8wxXrSU6aB3gxoA9wZ3fYTlspoEtIaaQl5wlJFWs3WYymo7y9pyMAsllXZqUliZu63LK3vpiZpb7Tr+EwXIvIa0mYWLzW597J/aGnXKCBsWp/oGwN5Klo2SdO09h/EeorVyhM/YLtZV0f1JzeFHeXgPTZ9mSunCvBARHMV/Y2DnpsNX9W7dHVDw/QuYU2pSXy+IqjG0ng8LQ6mYGN6G4Rk9qjRSCz6Ax7eOei/AVGsiHnzeUFP/bo+HG+fkMErGS0TmPf0aNlYdSunzvpwxOk1OdgedZ3tWNOa/M2Sg=
+  val: eyJ0aW1lc3RhbXAiOjE1MjQ0Mjc3NjgxNjEsInByb2ZpbGVJZCI6ImVmN2JjYzk2M2FjMDRkYzZhYWU3MDE5ZTVjNmRkYTJlIiwicHJvZmlsZU5hbWUiOiJzdXBlcmlvcnhtYW4iLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlL2NiMjEyYjFjMGU0NzQ5YjljN2E5ZmZhOGYyYThlMzc5NjlmYzk1YzY3ZDk4ZTg3ZWY5ZmM0YWY1YWZhNjgifX19
```

Comment:
Please look at how beast boy's multiple skins are made, you are missing the dashes